p2p/test/p2ptest.go

```diff
@@ -38,17 +37,17 @@ func NewTestNetwork(ctx context.Context, t *testing.T, num int, maker NodeMakerF
 	nodes := make([]TestablePeerNode, num)
 
 	for i := 0; i < num; i++ {
-		rid, err := testutil.RandPeerID()
-		if err != nil {
-			return nil, fmt.Errorf("error creating peer ID: %s", err.Error())
-		}
-
-		r, err := test.NewTestRepoFromProfileID(profile.ID(rid), i, i)
+		// TODO: This is ugly. Needed because p2p/connected_test generates 5 nodes with
```

Comment:
👌 long term I think we should rethink the way we generate test networks, but _that_ is a story for another day (and RFC)